server/src/main/java/org/apache/iotdb/db/metadata/mtree/traverser/counter/MeasurementCounter.java

```diff
@@ -43,6 +62,11 @@ protected boolean processFullMatchedMNode(IMNode node, int idx, int level) {
     if (!node.isMeasurement()) {
       return false;
     }
+    if (hasTag) {
+      if (!timeseries.contains(node.getFullPath())) {
+        return true;
+      }
+    }
     count++;
     return true;
   }
```

Comment:
if haveTag=true, here return, the count will not be calculated?
the same as others